aws/table_aws_ssm_document_permission.go

```diff
@@ -0,0 +1,133 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	"github.com/aws/aws-sdk-go-v2/service/ssm/types"
+
+	ssmv1 "github.com/aws/aws-sdk-go/service/ssm"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsSSMDocumentPermission(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ssm_document_permission",
+		Description: "AWS SSM Document Permission",
+		List: &plugin.ListConfig{
+			Hydrate: listAwsSSMDocumentPermissions,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "document_name", Require: plugin.Required},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(ssmv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "document_name",
+				Description: "The name of the Systems Manager document.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "shared_account_id",
+				Description: "The Amazon Web Services account ID where the current document is shared.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("AccountSharingInfo.AccountId"),
+			},
+			{
+				Name:        "shared_document_version",
+				Description: "The version of the current document shared with the account.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("AccountSharingInfo.SharedDocumentVersion"),
+			},
+			{
+				Name:        "account_ids",
+				Description: "The account IDs that have permission to use this document.The ID can be either an AWS account or All.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
```

Comment:
@ParthaI Is there a specific reason why we did not add other standard columns?

Reply:
@misraved At first AWS does not treat `aws_ssm_document_permission` as a resource type.
As per the steampipe plugin table development standard if any hydrate data supports pagination we considered it as a separate table. Also the API response doesn't contain any tag information. So that I have not added any other standard column